llvm/lib/CodeGen/RegAllocGreedy.cpp

```diff
@@ -2688,6 +2707,30 @@ bool RAGreedy::runOnMachineFunction(MachineFunction &mf) {
   if (VerifyEnabled)
     MF->verify(this, "Before greedy register allocator");
 
+  // getAnalysis<SimpleAnalysis>().printmf(*MF);
+
+  std::vector<TensorSpec> FeatureList {
```

Comment:
make this guy a member, then the for loop at line 306 just needs to go over the FeatureList instead of up to "2"